packages/site/src/components/HeaderBlock.tsx

```diff
@@ -33,17 +39,16 @@ export const HeaderBlock = ({
         className="headerBlock"
       >
         <Content spacing="large">
-          <Typography
-            element={"h1"}
-            size="extravagant"
-            fontWeight="bold"
-            fontFamily="display"
-          >
-            {title}
-          </Typography>
-          <Typography size="large" fontWeight={"semiBold"}>
-            {body}
-          </Typography>
+          <div>
```

Comment:
Slightly annoying detail for anyone interested in improving `AtlantisThemeContextProvider` - requires an additional wrapper to be placed outside of itself for `Content` to work, because the `margin` applied by Content to its direct children is subsumed by the `AtlantisThemeContextProvider`s `display: contents`

Reply:
(Still fewer LOC than the Typography solution, though 😄)

Reply:
We should definitely document this somewhere. Maybe bring it in the channel for thoughts first